src/v2/guide/migration.md

```diff
@@ -1427,38 +1427,44 @@ vm.$el.remove()
 </div>
 {% endraw %}
 
-## Meta Instance Methods
+## Métodos de metainstancia
```

Comment:
What do you think about changing `metainstancia` to `meta instancia` or `meta-instancia`?

Reply:
I like your suggestion, I'll do the change asap.